runtime/arc.js

```diff
@@ -629,19 +629,13 @@ ${this.activeRecipe.toString()}`;
     return this._storeDescriptions.get(store) || store.description;
   }
 
-  getStoresState() {
+  getStoresState(options) {
     let versionById = new Map();
     this._storesById.forEach((handle, id) => versionById.set(id, handle.version));
-    return versionById;
-  }
-
-  isSameState(storesState) {
```

Comment:
unrelated removal?

Reply:
it is a related method, that was never used, 
but now it is also wrong - only looking at the arc's stores, not the context, so ...

Reply:
just checking it was a deliberate removal :)